### PR TITLE
sci-physics/vgm: add 5.2

### PR DESCRIPTION
**Pull request by olifre**
In addition to adding the new release 5.2, it also:
* ~~drops the oldest release in-tree~~ (=> actually, don't, since some `sci-physics/geant4_vmc` versions for older `sci-physics/geant` depend on that)
* reactivates previously disabled tests for `ScaledSolids` in the live ebuild (this has been fixed upstream).

### CODE REVIEW

File: sci-physics/vgm/vgm-5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc examples +geant4 +root test"
+
+RDEPEND="
+	sci-physics/clhep:=
+	geant4? ( >=sci-physics/geant-4.11:=[c++17] )
+	root? ( sci-physics/root:=[c++17] )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[dot] )
```
Comment (thesamesam) [inline sci-physics/vgm/vgm-5.2.ebuild:30]: doxygen is a BDEPEND here (its tool is just called during the build).

Comment (olifre) [reply sci-physics/vgm/vgm-5.2.ebuild]: Great catch, thanks! Fixed in the last force-push both for the new version and `9999`. 
It's probably not useful backporting this fix to the older versions in-tree, or is it?

Comment (thesamesam) [reply sci-physics/vgm/vgm-5.2.ebuild]: No problem! Nah, I wouldn't worry about it.

**Comment by amadio** [comment]
Sorry for taking so long!

**Pull request closed without merging**